include/core/platform/lf_STM32f4_support.h

```diff
@@ -0,0 +1,23 @@
+/* STM32 API support for the C target of Lingua Franca. */
+
+#ifndef LF_STM32F4_SUPPORT_H
+#define LF_STM32F4_SUPPORT_H
+
+// I have no idea what the fuck TTY is so i guess we dont support it
+#define NO_TTY
+
+#include <stm32f4xx_hal.h>
+
+// Defines for formatting time in printf for pico
+#define PRINTF_TAG "(" PRINTF_TIME ", " PRINTF_MICROSTEP ")"
+#define PRINTF_TIME "%lld"
+#define PRINTF_MICROSTEP "%d"
+
+#define LF_TIME_BUFFER_LENGTH 80
+#define _LF_TIMEOUT 1
```

Comment:
What units is this #def? Should it be a const instead?

Reply:
I kinda just copied this over when I compared it against the rp2040 platform - I wasnt sure what this was used for so I kept it just incase. If its not used in LF proper, ill just delete it